src/manager/jobSeeker/jobSeeker.ts

```diff
@@ -0,0 +1,18 @@
+import { IJobSeekerProps } from '@allTypes/reduxTypes/areaSpecializationTypes'
+import { Axios } from '@axios/axiosInstance'
+import { IAxiosResponse } from '@axios/axiosTypes'
+
+const axiosInstance = Axios()
+const jobSeekerManager = {
+  axiosInstance,
+  jobSeeker(data: IJobSeekerProps) {
+    return axiosInstance.post<IJobSeekerProps, IAxiosResponse<IJobSeekerProps>>(
+      `users/v1/job-seeker/application`,
+      {
+        ...data,
+      }
+    )
+  },
```

Comment:
Jobseeker is not manager rename to applications

Reply:
const baseURL = '/users'

Reply:
done